content/developer/reference/standard_modules/account/account_account.rst

```diff
@@ -0,0 +1,18 @@
+.. _reference/account_account:
+
+=======
+Account
+=======
+
+.. autoodooclass:: odoo.addons.account.models.chart_template.AccountAccountTemplate
+    
+    .. autofield:: name
+    .. autofield:: currency_id
+    .. autofield:: code
+    .. autofield:: user_type_id
+    .. autofield:: reconcile
+    .. autofield:: note
+    .. autofield:: tax_ids
+    .. autofield:: nocreate
+    .. autofield:: chart_template_id
+    .. autofield:: tag_ids
```

Comment:
Do these have to be listed as autofields, can't `:members:` do the job? That would make the *auto* part of autoodooclass a lot truer.

Reply:
`:members:` is not that good of an idea to me because
* we rarely want all the fields (we don't care about computed and technical fields in this API)
* I like to have one field per line for VCS reasons

Reply:
> `:members:` is not that good of an idea to me because
> 
>  * we rarely want all the fields (we don't care about computed and technical fields in this API)

That can be handled through selection flags in exactly the way autodoc normally works, and `:members:` supports whitelisting.

> * I like to have one field per line for VCS reasons

:shrug:

Reply:
Ok, but what's the difference, apart from more development? If we always need to whitelist, then what's the point?